.github/workflows/publish.yml

```diff
@@ -0,0 +1,87 @@
+name: Wheel building
+
+on:
+  schedule:
+    # run every day at 4am UTC
+    - cron: '0 4 * * *'
+  workflow_dispatch:
+  push:
+  pull_request:
+
+jobs:
+
+  test_wheel_building:
+    # This ensures that a couple of targets work fine in pull requests and pushes
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@v1
+    if: (github.event_name == 'push' || github.event_name == 'pull_request')
+    with:
+      upload_to_pypi: false
+      upload_to_anaconda: false
+      test_extras: test
+      test_command: pytest -p no:warnings --astropy-header -m "not hypothesis" -k "not test_data_out_of_range and not test_wcsapi_extension and not test_set_locale" --pyargs astropy
+      targets: |
+        - cp39-manylinux_x86_64
+
+  build_and_publish:
+    # This does the actual wheel building and publishing as part of the cron job
+    # or if triggered manually via the workflow dispatch.
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@v1
+    if: (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch')
```

Comment:
Do we need to add `github.repository == 'astropy/astropy'` here to avoid forks with Actions enabled running this?

https://github.com/astropy/astropy/blob/f0e2129aad460ea0db00730f10f98fe799aefb2f/.github/workflows/ci_cron_weekly.yml#L33

Reply:
Ah yes good catch!